downstream/modules/aap-hardening/con-rhel-host-planning.adoc

```diff
@@ -3,10 +3,10 @@
 
 [id="con-rhel-host-planning_{context}"]
 
-= RHEL host planning
+= Red Hat Enterprise Linux host planning
 
 [role="_abstract"]
 
-The security of {PlatformNameShort} (AAP) relies in part on the configuration of the underlying Red Hat Enterprise Linux (RHEL) servers. For this reason, the underlying RHEL hosts for each AAP component must be installed and configured in accordance with the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/security_hardening/index[RHEL 8 Security Hardening Guide] or link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/security_hardening/index[RHEL 9 Security Hardening Guide] (depending on which operating system will be used), as well as any security profile requirements (CIS, STIG, HIPAA, etc.) used by your organization.
+The security of {PlatformNameShort} relies in part on the configuration of the underlying Red Hat Enterprise Linux servers. For this reason, the underlying Red Hat Enterprise Linux hosts for each {PlatformNameShort} component must be installed and configured in accordance with the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/security_hardening/index[RHEL 8 Security Hardening Guide] or link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/security_hardening/index[RHEL 9 Security Hardening Guide] (depending on which operating system will be used), as well as any security profile requirements (CIS, STIG, HIPAA, and so on) used by your organization.
```

Comment:
Edit link titles: 
"[RHEL 8 Security Hardening Guide]" to "[Security hardening for Red Hat Enterprise Linux 8]"
"[RHEL 9 Security Hardening Guide]" to "[Security hardening for Red Hat Enterprise Linux 9]"